packages/backend/tsconfig.index.json

```diff
@@ -0,0 +1,20 @@
+{
+    // This file will extend the tsconfig.json file for the test environment
+    // This means we will not apply the noUncheckedIndexedAccess option to the test files
+    // Change build method in package.json to use this file
+    // It is temporary disabled in package.json until we fix all the errors
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "noUncheckedIndexedAccess": true
+    },
+    "include": [
+        "src/**/*.ts",
+        "src/**/*.json"
+    ],
+    "exclude": [
```

Comment:
Adding the files to exclude will exclude them from all the tsconfig rather than just that rule 

How I tested this:
- I added an unused variable to a test file, which is excluded in my noUncheckedIndexedAccess file
- I don't get an error,
- I remove the `        "src/*/.test.ts",` exclusion from my extra file
- I get the error now